README.rst

```diff
@@ -110,7 +116,7 @@ To build the PHP driver for Windows:
        git clone https://github.com/snowflakedb/pdo_snowflake.git
        cd pdo_snowflake
 
-#. Run the script to download the PHP SDK:
+#. Run the script to download the PHP source code:
```

Comment:
This step should be unchanged as it is downloading PHP SDK

Reply:
But the next step: `Download and install the PHP binaries, or build PHP yourself.` should be changed though, like 'Download and build PHP source code'

Reply:
Done